knowrob_assembly/prolog/init.pl

```diff
@@ -34,7 +34,7 @@
 :- register_ros_package(knowrob_objects).
 :- register_ros_package(knowrob_planning).
 
-:- use_module(library('knowrob_assembly')).
-:- use_module(library('knowrob_battat')).
+:- use_module('knowrob_assembly').
```

Comment:
would merge without this change.
Looks a bit like `register_ros_package(knowrob_assembly)` was not called before.
Could you please  try to add this above, and see if `use_module(library('knowrob_assembly'))` works then ?